format/Flight.proto

```diff
@@ -124,11 +124,6 @@ message HandshakeRequest {
    * A defined protocol version
    */
   uint64 protocol_version = 1;
-
```

Comment:
These are format changes and will need a vote.

Reply:
Thanks for the suggestions @lidavidm .

Reply:
Format changes are gone now.